src/containers/GetAnArticle.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import AnArticle from '../components/AnArticle';
+import Comments from '../components/Comments';
+import {
+  fetchAnArticle,
+  fetchArticleComments,
+} from '../store/actions/commentsActions';
+
+export class GetAnArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.slug = '';
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    this.slug = match.params;
+    const { fetchAnArticle } = this.props;
+    const { fetchArticleComments } = this.props;
+    fetchAnArticle(this.slug);
+    fetchArticleComments(this.slug);
+  }
+
+  render() {
+    const articleRender = () => {
+      const { isFetchingArticle } = this.props;
+      const { errors } = this.props;
+      const { isFetchingComments } = this.props;
+      if (isFetchingArticle) {
+        return (<div className="article-loading">Loading...</div>);
+      }
+      if (errors) {
+        return (<div className="article-errors">Return 404...</div>);
+      }
+      if (isFetchingComments) {
+        return (<div className="comments-loading">Loading...</div>);
+      }
+      const { article } = this.props;
```

Comment:
destructure these in one line, for example
```suggestion
      const { article, comments } = this.props;
```